tests/operator/operator.go

```diff
@@ -1361,7 +1345,11 @@ spec:
 			}
 
 			By("starting a VM")
-			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+			vmi := libvmi.New(libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+				libvmifact.WithCirrosRequestedMemory(),
+				libvmi.WithContainerDisk("disk0", cd.ContainerDiskFor(cd.ContainerDiskCirros)))
```

Comment:
Looks to be equivalent to `vmi := libvmi.NewCirros()`
All other setting are irrelevant to the test or included in the factory.